entries/m/myfritz.net.json

```diff
@@ -0,0 +1,20 @@
+{
+  "MyFRITZ!Net": {
+    "domain": "myfritz.net",
+    "url": "https://www.myfritz.net",
```

Comment:
```suggestion
```

The `url` key is only needed if going to the domain (e.g. `https://myfritz.net`) fails